site/pages/xprmntl/checkboxes-en.hbs

```diff
@@ -0,0 +1,338 @@
+	<li>For smaller checkboxes (tickboxes, the class <code>gc-chckbxrdio</code> has to be enabled on the container element surrounding the targeted checkbox and the class <code>checkbox-small</code> has to be applied to each checkbox input element.</li>
+	<li>To toggle a disabled state, add a <code>disabled</code> attribute to the input field.</li>
+	<li>For inline form, add the <code>form-inline</code> class to surrounding container of the checkboxes or radio buttons. Also add the <code>label-inline</code> class to each checkbox or radio button container.</li>
+</ul>
+<h4><code>.gc-chckbxrdio</code></h4>
+<h5>Three checkboxes layout</h5>
+<pre><code>&lt;fieldset class="provisional gc-chckbxrdio chkbxrdio-grp"&gt;
+	&lt;legend&gt;[[comp:Checkboxes dct:title]]:&lt;/legend&gt;
+	&lt;ul class="list-unstyled lst-spcd-2"&gt;
+		&lt;li class="checkbox"&gt;
+			&lt;input type="checkbox" id="[[comp:ChckbxrdioItem html:input-id]]"&gt;
+			&lt;label for="[[comp:ChckbxrdioItem html:input-id]]"&gt;[[comp:ChckbxrdioItem in (comp:Chckbxrdio comp:Chckbxsrdios )]]&lt;/label&gt;
+		&lt;/li&gt;
+		&lt;li class="checkbox"&gt;
+			&lt;input type="checkbox" id="[[comp:ChckbxrdioItem html:input-id]]"&gt;
+			&lt;label for="[[comp:ChckbxrdioItem html:input-id]]"&gt;[[comp:ChckbxrdioItem in (comp:Chckbxrdio comp:Chckbxsrdios )]]&lt;/label&gt;
+		&lt;/li&gt;
+		&lt;li class="checkbox"&gt;
+			&lt;input type="checkbox" id="[[comp:ChckbxrdioItem html:input-id]]"&gt;
+			&lt;label for="[[comp:ChckbxrdioItem html:input-id]]"&gt;[[comp:ChckbxrdioItem in (comp:Chckbxrdio comp:Chckbxsrdios )]]&lt;/label&gt;
+		&lt;/li&gt;
+	&lt;/ul&gt;
+&lt;/fieldset&gt;</code></pre>
+
+<h4>Features Variant: <code>.checkbox-small</code></h4>
+<pre class="mrgn-tp-md"><code>&lt;form class="form-inline" method="get" action="#"&gt;
+	&lt;div class="form-group"&gt;
+		&lt;label class="wb-inv" for="exampleInputEmail2"&gt;Email address&lt;/label&gt;
+		&lt;input type="email" class="form-control" id="exampleInputEmail2" placeholder="Enter email" /&gt;
+	&lt;/div&gt;
+	&lt;div class="form-group"&gt;
+		&lt;label class="wb-inv" for="exampleInputPassword2"&gt;Password&lt;/label&gt;
+		&lt;input type="password" class="form-control" id="exampleInputPassword2" placeholder="Password" /&gt;
+	&lt;/div&gt;
+	&lt;div class="checkbox provisional gc-chckbxrdio"&gt;
+		&lt;input id="[[comp:CheckboxItem html:input-id]]" type="checkbox" class="checkbox-small"/&gt;
+		&lt;label for="[[comp:CheckboxItem html:input-id]]"&gt;[[comp:CheckboxItem in (comp:Checkbox comp:checkboxes )]]&lt;/label&gt;
+	&lt;/div&gt;
+	&lt;button type="submit" class="btn btn-default"&gt;Sign in&lt;/button&gt;
+&lt;/form&gt;</code></pre>
+<h3>Visual rendering (v1.0)</h3>
+<details>
+	<summary>Version notes</summary>
+	<dl class="dl-horizontal">
+		<dt>Version 1.0</dt>
+		<dd>
+			<ul>
+				<li>Original rendering</li>
+			</ul>
+		</dd>
+	</dl>
+</details>
+<h3>Schema (v1.0)</h3>
```

Comment:
You can remove the schema, this template/pattern is more a style rather than a complex component (ex. doormat link)